diff --git include/czmq_prelude.h include/czmq_prelude.h
index 9d2c2f9..a323ee8 100644
--- include/czmq_prelude.h
+++ include/czmq_prelude.h
@@ -392,7 +392,6 @@ typedef unsigned int    qbyte;          //  Quad byte = 32 bits
 //- A number of POSIX and C99 keywords and data types -----------------------
 
 #if (defined (__WINDOWS__))
-#   define inline __inline
 #   define strtoull _strtoui64
 #   define srandom srand
 #   define TIMEZONE _timezone
