diff -r dd5a807cd5ec CMakeLists.txt
--- a/CMakeLists.txt	Sat Sep 28 19:23:59 2013 -0300
+++ b/CMakeLists.txt	Mon Sep 30 06:53:32 2013 -0700
@@ -1215,11 +1215,13 @@
 list(APPEND INCLUDE_FILES ${BIN_INCLUDE_FILES})
 install(FILES ${INCLUDE_FILES} DESTINATION include/SDL2)
 
-if(SDL_SHARED)
-  install(CODE "
-    execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink
-    \"libSDL2-2.0.so\" \"libSDL2.so\")")
-  install(FILES ${SDL2_BINARY_DIR}/libSDL2.so DESTINATION "lib${LIB_SUFFIX}")
+if(SDL_SHARED AND NOT WINDOWS)
+   install(CODE "
+     execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink
+     \"libSDL2-2.0.so\" \"libSDL2.so\")")
+   install(FILES ${SDL2_BINARY_DIR}/libSDL2.so DESTINATION "lib${LIB_SUFFIX}")
+elseif(WINDOWS)
+  install(TARGETS SDL2 RUNTIME DESTINATION bin)
 endif(SDL_SHARED)
 
 if(NOT WINDOWS OR CYGWIN)
